compile-solidity.sh

```diff
@@ -12,9 +14,9 @@ solc-err-only --overwrite --optimize --bin --abi BuyNowOfferingFactory.sol -o ..
 solc-err-only --overwrite --optimize --bin --abi AuctionOfferingFactory.sol -o ../build/
 solc-err-only --overwrite --optimize --bin --abi District0xEmails.sol -o ../build/
 solc-err-only --overwrite --optimize --bin --abi OfferingRequests.sol -o ../build/
-solc-err-only --overwrite --optimize --bin --abi ens-repo/contracts/ENS.sol -o ../build/
-solc-err-only --overwrite --optimize --bin --abi ens-repo/contracts/PublicResolver.sol -o ../build/
-solc-err-only --overwrite --optimize --bin --abi ens-repo/contracts/ReverseRegistrar.sol -o ../build/
+solc-err-only --overwrite --optimize --bin --abi $ENS/ens/contracts/ENS.sol -o ../build/
```

Comment:
Oh, nice. I didn't notice that they publish the contract sources in the npm package as well.